app/elements/socobo-element-search/socobo-element-search.html

```diff
@@ -163,6 +163,17 @@
         searchQueryChanged: function() {
           this.fire("changed-search-query", { searchQuery: this.searchStr });
         },
+
+        /**
+         * Method for resetting the search query </br>
+         * @method resetQuery
+         */
+        resetQuery: function() {
+          this.searchStr = "";
+          this.searchQueryChanged();
+          this.$.chbCheckAllItems.checked = false;
+          this.fire("uncheck-all-items");
```

Comment:
If there is no further application for this method than after deleting items, the checked items are gone anyway and firing this event serves no purpose anymore.

But maybe firing this event makes deselecting stuff easier, preventing accidentally leaving items checked.